opus/application/static_media/js/opus.js

```diff
@@ -390,7 +386,7 @@ var opus = {
             $("#op-help-panel .card-body").scrollTop(0);
             opus.helpScrollbar.update();
         }
-    }
+    },
```

Comment:
Does this comma serve a purpose?

Reply:
It's removed now. It was there because there was another function down there in the same namespace.